src/web/components/SignInGate/gateDesigns/SignInGateMain.tsx

```diff
@@ -250,7 +250,7 @@ export const SignInGateMain = ({
             </div>
             {showCpmUi && (
                 <Suspense fallback={<></>}>
-                    <ConsentManagementPlatform
+                    <oldCmp.ConsentManagementPlatform
```

Comment:
@guardian/dotcom-platform I'm not sure what to do about these – does this sign in gate component appear in place of `StickyBottomBanner.tsx`?

Reply:
so we do need to add support for v4 in here, confirmed